.eslintrc

```diff
@@ -25,18 +26,18 @@
         "indent": [2, 4, {
             "SwitchCase": 1
         }],
+        // require 4 spaces in JSX as well
+        "react/jsx-indent": [0],
+        "react/jsx-indent-props": [0],
```

Comment:
Why are we turning this off? And if justified, remove/update the comment associated with it.

Reply:
It's getting turned off because it seems to act oddly suddenly and looks for an extra tab where there shouldn't be one. It's in direct conflict with the regular indent, which covers all files (tested locally by adding an extra space in a file and it showed up in the linter even with these turned off). The current status, including comments, directly mirrors the USAS linter setting file. Did we want to deviate from that?